daphne/src/messages/mod.rs

```diff
@@ -15,6 +15,8 @@ use std::{
     io::{Cursor, Read},
 };
 
+use crate::{DapError, DapVersion};
```

Comment:
nit: If you remove the line between this `use` and the previous one, then your linter should automatically put it in alphabetical order.